clang/lib/Sema/SemaLambda.cpp

```diff
@@ -1494,6 +1494,14 @@ static void addFunctionPointerConversion(Sema &S, SourceRange IntroducerRange,
   Conversion->setAccess(AS_public);
   Conversion->setImplicit(true);
 
+  // Add AsmJS attribute to function object if it exists on the lambda
```

Comment:
Looks good. Just add a `// CHEERP:` in front of the comment, to make it easier to find code that we added

Reply:
Done!